src/ispc.h

```diff
@@ -811,6 +823,9 @@ struct Globals {
         vector width to them. */
     bool mangleFunctionsWithTarget;
 
+    /** If enabled, it means that we generate code only for one target. */
+    bool singleTargetCompilation;
```

Comment:
Not sure if static analysis will be OK with not-initializing this property in `Globals` constructor.

Reply:
Actually, I realized that there is `isMultiTargetCompilation` and removed `singleTargetCompilation`.